app/components/ui/Dashboard.tsx

```diff
@@ -1,12 +1,14 @@
 import React, { useRef, useState } from "react";
 import GraphCard from "./GraphCard";
 import { Household } from "@/app/models/Household";
+import { formType } from "@/app/schemas/formSchema";
```

Comment:
```suggestion
import type { Form } from "@/app/schemas/formSchema";
```

Types and interfaces should be in PascalCase, and if we can avoid it we shouldn't use `Type` suffixes.